scheduler/kubernetes/ark.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: "Using Ark with Portworx"
+keywords: portworx, container, Kubernetes, storage, k8s, pv, persistent disk, snapshot
+sidebar: home_sidebar
+---
+
+* TOC
+{:toc}
+
+Heptio Ark is a utility for managing disaster recovery, specifically for your
+Kubernetes cluster resources and persistent volumes. To take snapshots of
+Portworx volumes through Ark you need to install and configure the Portworx
+plugin.
+
+## Install Ark Plugin
+Run the following command to install the Portworx plugin for Ark:
+```
+ark plugin add portworx/ark-plugin:0.1
+```
+
+This should add an init container to your Ark deployment to install the
+plugin.
+
+## Configure Ark to use Portworx snapshots
+
+Once the plugin is installed you need to configure Ark to use Portworx as the
+Persistent Volume Provider when taking snapshots. To edit the config run the
+following command:
+
+```
+kubectl edit config -n heptio-ark
+```
+
+And set up `portworx` as the `persistentVolumeProvider` by adding the following
+snippet to the config spec:
+```
+persistentVolumeProvider:
+  name: portworx
+```
+
+## Taking snapshots
+Once the plugin has been installed and configured, everytime you take backups
+using Ark and include PVCs, it will also take Portworx snapshots of your volumes.
+When restoring from backups, a clone volume will be created from the snapshot and
+bound to the restored PVC.
```

Comment:
Can you put a couple of lines on how this works under the hood? This will save time in explaining it to end users.

Reply:
Added